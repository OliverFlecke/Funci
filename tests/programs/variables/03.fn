main = let nil = []
                 in let cons3nil = 3 : nil
                     in cons3nil
