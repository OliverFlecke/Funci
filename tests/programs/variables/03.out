Listy (Cons (Number 3 (Unit [])) Empty)
